opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -1283,6 +1283,27 @@ def test_record_exception_context_manager(self):
         finally:
             self.assertEqual(len(span.events), 0)
 
+    def test_record_exception_out_of_scope(self):
+        span = trace._Span("name", mock.Mock(spec=trace_api.SpanContext))
+        try:
+            raise ValueError("invalid")
+        except ValueError as err:
+            out_of_scope_exception = err
```

Comment:
Does this behave differently than just using the value error as a variable?
```suggestion
        out_of_scope_exception = ValueError("invalid")
```

Reply:
Need to check